style.css

```diff
@@ -1,17 +1,26 @@
 @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&family=Space+Mono&display=swap');
 
+* {
+  padding: 0;
+}
+
+body {
+  background-color: #e5e5e5;
+}
+
 .headline {
   background-image: url(./images/header-shapes-mobile.png);
   background-repeat: no-repeat;
   background-size: cover;
   display: flex;
   flex-direction: column;
-  overflow-y: scroll;
-  -webkit-overflow-scrolling: touch;
+  background-color: #fff;
+  border-bottom-left-radius: 60px;
+  padding-bottom: 64px;
 }
 
 .nav-bar {
-  background-color: transparent;
+  background-color: white;
   margin-top: 44px;
```

Comment:
This suggestion is for avoiding extra spacing in the first section of this page. You don't need margin-top here.
```suggestion
```

Reply:
Made all changes.Thank you for your review!